src/Cache/CacheEngine.php

```diff
@@ -45,7 +45,8 @@
         'duration' => 3600,
         'groups' => [],
         'prefix' => 'cake_',
-        'probability' => 100
+        'probability' => 100,
+        'warnOnFailures' => true,
```

Comment:
IMO this should be false by default. For e.g. most people won't care about failures when calling `clear()`.

Reply:
For now I opted to keep this on by default but I removed almost all warnings, except those where writes cache fail which is not supposed to happen, right?